src/Components/People.js

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+import { Component } from "react";
+
+class People extends Component {
+  state = {
+    people: [],
+    id: "",
+    person:{},
+    name:"",
+    age:"",
+    gender:""
+  };
+
+  handleForm = async(e) =>{
+      e.preventDefault()
+      const {person} = this.state
+      if(person[0]){
+      this.setState({
+        name: person[0].name,
+        age: person[0].age,
+        gender: person[0].gender
+      })}else{
+          this.setState({
+              name:"Not Found",
+              age:null,
+              gender:null
+            })
+
+      }
+    
+debugger
+  }
+
+  handleChange = async(e) => {
```

Comment:
This should be happening inside the handle submit. There's no need to make this filter check every time a user types an individual letter